calico/README.md

```diff
@@ -0,0 +1,155 @@
+
+Using annotations:
+
+1. Set up Calico on your cluster if you have not already.
+
+2. Enable Prometheus metrics using the instructions in [Monitor Calico component metrics][9].
+   Once enabled, you should have a `felix-metrics-svc` service running in your cluster, as well as a `prometheus-pod`.
+
+3. To use Autodiscovery, modify `prometheus-pod`. Add the following snippet to your Prometheus YAML configuration file:
+
+```
+metadata:
+  [...]
+  annotations:
+   ad.datadoghq.com/prometheus-pod.check_names: |
+   ["openmetrics"]
+   ad.datadoghq.com/prometheus-pod.init_configs: |
+   [{}]
+   ad.datadoghq.com/prometheus-pod.instances: |
+     [
+        {
+           "prometheus_url": "http://<FELIX-SERVICE-IP>:<FELIX-SERVICE-PORT>/metrics",
+           "namespace": "calico",
+           "metrics": ["*"]
+        }
+     ]
+  spec:
+    [....]
+```
+
+You can find values for `<FELIX-SERVICE-IP>` and `<FELIX-SERVICE-PORT>` by running `kubectl get all -all-namespaces`.
+
+#### Installation with an OS-based Agent
+
+1. Follow [Monitor Calico component metrics][9] until you have a `felix-metrics-svc` service running using `kubectl get all --all-namespaces`.
+
+2. If you are using minikube, you must forward port 9091 to `felix-metrics-svc`.
+   Run `kubectl port-forward service/felix-metrics-svc 9091:9091 -n kube-system`.
+
+   If you are not using minikube, check that `felix-metrics-svc` has an external IP. If the service does not have an external IP, use `kubectl edit svc` to change its type from `ClusterIP` to `LoadBalancer`.
+
+Once installation is complete, you can continue to configuration (see below).
+
+### Configuration
+
+
+Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized](#containerized) section.
+
+<!-- xxx tabs xxx -->
+<!-- xxx tab "Host" xxx -->
+
+#### Host
+
+To configure this check for an Agent running on a host:
+
+1. Edit the `calico.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your Calico performance data.
+   The only required parameter is the `openmetrics_endpoint` URL. See the [sample calico.d/conf.yaml][3] for all available configuration options.
+
+2. If you are using minikube, use 'http://localhost:9091/metrics' as your `openmetrics_endpoint` URL.
+   If you are not using minikube, use `http://<FELIX-METRICS-SVC-EXTERNAL-IP>:<PORT>/metrics` as your `openmetrics_endpoint` URL.
+
+3. [Restart the Agent][4].
+
+##### Metric collection
+
+1. The default configuration of your `cassandra.d/conf.yaml` file activate the collection of your [Calico metrics](#metrics). See the [sample calico.d/conf.yaml][3] for all available configuration options.
+
+2. [Restart the Agent][4].
+
+##### Log collection
+
+Since Calico structure is set up in a Kubernetes cluster, it is built with deployments, pods, and services.
+The Kubernetes integration fetches logs from containers.
```

Comment:
```suggestion
Since Calico structure is set up in a Kubernetes cluster, it is built with deployments, pods, and services. The Kubernetes integration fetches logs from containers.

```